front/src/modules/trainschedule/components/Timetable/PacedTrain/PacedTrainItem.tsx

```diff
@@ -297,6 +299,16 @@ const PacedTrainItem = ({
       )}
     </div>
   );
+
+  if (!isOnEdit) {
+    return content;
+  }
+
+  const container = document.getElementById('timetable-edited-element');
```

Comment:
This will not work if React re-renders this element.

Why do we need to create a portal here?

Reply:
done

Reply:
Hm, I'm not in love with the way this works, but oh well. Hopefully this can be all dropped once the new train creation workflow is implemented.

Reply:
I share your opinion. we were showing an element hidden behind the left panel, completely out of the page flow. kinda a workaround of a workaround i had to do here, no need to invest more time here